kfs-core/src/main/java/edu/arizona/kfs/gl/businessobject/lookup/GecEntryHelperServiceImpl.java

```diff
@@ -106,19 +108,19 @@ public static String generateObjectId(GecEntry entry) {
      */
     public static Entry getEntry(String objectId) {
         String[] primaryKeyFields = objectId.split(SecConstants.SecurityValueSpecialCharacters.MULTI_VALUE_SEPERATION_CHARACTER);
-        String universityFiscalYear = primaryKeyFields[0];
-        String chartOfAccountsCode = primaryKeyFields[1];
-        String accountNumber = primaryKeyFields[2];
-        String subAccountNumber = primaryKeyFields[3];
-        String financialObjectCode = primaryKeyFields[4];
-        String financialSubObjectCode = primaryKeyFields[5];
-        String financialBalanceTypeCode = primaryKeyFields[6];
-        String financialObjectTypeCode = primaryKeyFields[7];
-        String universityFiscalPeriodCode = primaryKeyFields[8];
-        String financialDocumentTypeCode = primaryKeyFields[9];
-        String financialSystemOriginationCode = primaryKeyFields[10];
-        String documentNumber = primaryKeyFields[11];
-        String transactionLedgerEntrySequenceNumber = primaryKeyFields[12];
+        String universityFiscalYear = primaryKeyFields[1];

```

Comment:
The reason why is because the first field is in the split (index 0) is to identify the object Id as being for Entry objects. This was needed for GeneralErrorCorrectionAction because this is a second multi-value lookup that is on the document. Without this change, the GECAction would apply the Entry logic to the Asset lookup, which causes STEs.